presto-docs/src/main/sphinx/connector/hive-s3.rst

```diff
@@ -158,6 +158,15 @@ For example, the mapping list might have URL prefix ``s3://abc/xyz/`` followed b
 ``s3://abc/`` to allow different configuration for a specific path within a bucket
 than for other paths within the bucket. You can set default configuration by not
 including any match criteria for the last entry in the list.
+In addition to rules above the default mapping could contain
```

Comment:
Start a new paragraph for this, so add empty line above. Also 

```
In addition to the rules above, the default mapping can contain the ``userClusterDefault`` property set to ``true``. This causes the the default cluster role to be used as a fallback option. It can not be used with the  following configuration properties:

- ``accessKey``
...

```

Reply:
done.